### PR TITLE
fix: use root path in code

### PR DESCRIPTION
**Pull request by leehuwuj**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -31,6 +31,8 @@
 # Deployment files
 ./Dockerfile
 ./docker-compose.yml
+./deployments/**/data
```
Comment (leehuwuj) [inline .dockerignore:34]: to avoid copy large of data to the image when development

File: main.py [hunk 2]
```diff
@@ -24,7 +24,10 @@
 from src.routers.management.reranker import reranker_router
 from src.routers.management.tools import tools_router
 
-app = FastAPI()
+app = FastAPI(
+    title="RAGapp",
+    root_path=os.getenv("BASE_URL", ""),
```
Comment (leehuwuj) [inline main.py:29]: we can just simply specify root path here 😄

File: patch/frontend/next.config.mjs [hunk 3]
```diff
@@ -8,8 +8,10 @@ import webpack from "./webpack.config.mjs";
 
 const nextConfig = JSON.parse(fs.readFileSync("./next.config.json", "utf-8"));
 
-// This prefix will be rewritten by BASE_URL when deployed, see entrypoint.sh
-nextConfig.assetPrefix = "https://static-assets.ragapp.org";
+process.env.ENVIRONMENT === "dev"
```
Comment (leehuwuj) [inline patch/frontend/next.config.mjs:11]: Always set assetPrefix won't work for development.

**Review by marcusschiesser — APPROVED** [review]
Looks great! Will merge after testing

**Review by marcusschiesser — APPROVED** [review]

**Pull request merged**